frontend/components/CustomLink/CustomLink.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import Icon from "components/Icon";
+
+interface ICustomLinkProps {
+  url: string;
+  text: string;
+  /** Default opens in a new tab */
+  newTab?: boolean;
+  //* Icon wraps on new line with last word */
+  multiline?: boolean;
+}
+
+const baseClass = "custom-link";
+
+const CustomLink = ({
+  url,
+  text,
+  newTab = true,
+  multiline = false,
+}: ICustomLinkProps): JSX.Element => {
+  const target = newTab ? "_blank" : "";
+
+  if (multiline) {
+    const multilineText = text.substring(0, text.lastIndexOf(" ") + 1);
+    const lastWord = text.substring(text.lastIndexOf(" ") + 1, text.length);
+
+    return (
+      <a
+        href={url}
+        target={target}
+        rel="noopener noreferrer"
+        className={baseClass}
+      >
+        {multilineText}
+        <span className="no-wrap">
+          {lastWord}
+          <Icon name="external-link" />
+        </span>
```

Comment:
This bit can dynmically generated and rendered. Otherwise we have to ensure both anchor tags are kept up to day.

**something like this**




```tsx

const content = createLinkContent()

return (
    <a
      href={url}
      target={target}
      rel="noopener noreferrer"
      className={baseClass}
    >
      {content}
    </a>
  );

```
 this will remove the need to keep the anchor attributes up to date in two places.